### PR TITLE
Even with google single sign on it still requires user to use password

### PR DESCRIPTION
**Pull request by gauravsontakke**
Issue # https://github.com/openemr/openemr/issues/7254

#### Short description of what this resolves:

Even with google single sign on it still requires user to use password from openEMR while signing the encounter.

Google single sign on works perfectly with the login but when it comes to signing an encounter single sign on doesn't work. It requires openEMR user password.

#### Changes proposed in this pull request:
Added google sign in functionality on eSign popup window to sign encounter and forms using google sign in. 
Signing encounter is possible with both openEMR password as well google single sign on.

### CODE REVIEW

File: library/ESign/views/encounter/esign_form.php [hunk 1]
```diff
@@ -61,3 +81,16 @@
         
     </form> 
 </div>
+
+<!-- Google sign in for esign -->
+<?php if ($this->form->displayGoogleSignin) { ?>
+<script type="text/javascript">
+    let gsi = Object.create(GoogleSigin);
+    gsi.init("<?php echo $this->form->googleSigninClientID; ?>", {
```
Comment (bradymiller) [inline library/ESign/views/encounter/esign_form.php:89]: lets escape this for javascript context (note the outside " are removed):
```
gsi.init(<php echo js_escape($this->form->googleSigninClientID); ?>, {
```

**Comment by bradymiller** [comment]
hi @gauravsontakke , Thanks for the improvements. I only noted couple minor escaping issues. Is this testing well for you?

**Comment by gauravsontakke** [comment]
Hi @bradymiller , 

Thank you so much for your review. I would like to know whether we should be working on the issues or you ll fix while merging ?

Regards
Gaurav

**Comment by stephenwaite** [comment]
hi @gauravsontakke, please push the fixes to those issues that Brady noticed in the review, thank you

**Comment by bradymiller** [comment]
hi @gauravsontakke ,
Code looks great!
Just confirm that this is testing well for you (and then I will bring this into the codebase).

**Comment by gauravsontakke** [comment]
Hi @bradymiller , 

Yes code works for us. We have this in production. 

Thank you so much. 

We are working on other commits as well. 

Regards
Gaurav

**Comment by bradymiller** [comment]
Just brought this in and it will also be in the next 7.0.2 patch. thanks @gauravsontakke !

**Pull request merged**